src/IIIFPresentation/Core/Web/TypedPathTemplateOptions.cs

```diff
@@ -0,0 +1,54 @@
+namespace Core.Web;
+
+/// <summary>
+/// A collection of options related to path generation. This differs from <see cref="PathTemplateOptions"/> as this
```

Comment:
`This differs from <see cref="PathTemplateOptions"/> as this` comment isn't relevant here, looks like copy/paste from Protagonist?